approval_polls/templates/approval_polls/edit.html

```diff
@@ -90,6 +91,12 @@ <h5 style="color:#8B0000;"><b>Please ensure you've entered all choices correctly
         </div>
       {% endfor %}
       {% endwith %}
+      <div id='tag-input'>
```

Comment:
Could you move the Add choice `+` button above the tag entry field ? I think that way it will be displayed along with the choices and be more intuitive.

Reply:
Hi @balajisubr , I just noticed this - I think you might have missed out addressing this issue as it seems that the '+' button remains below the tags instead of the choices. Let me know if you want to discuss it further or please feel free to open another pull request for this.